recipes/pycontracts/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "PyContracts" %}
+{% set version = "1.8.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/4e/7a/0f79370e4e3a6741396d76d1f76586c2924bed049fb38597799b72a24081/{{ name }}-{{ version }}.tar.gz"
```

Comment:
This should be the predictable version of the URL. See the example recipe for how to construct this

Reply:
Agree, but I don't know what kind of hash is being used in the url.

Reply:
https://github.com/conda-forge/staged-recipes/blob/eba25a75697c49852ce325ef73e9298cd3fede1a/recipes/example/meta.yaml#L14

Reply:
Ah I was testing it before fixing the name casing. Looks like that should work.